games/minimal/mods/experimental/modchannels.lua

```diff
@@ -0,0 +1,15 @@
+--
+-- Mod channels experimental handlers
+--
+
+core.after(2, function()
```

Comment:
I don't like this. Surely it could be possible to queue the join somehow? At least with builtin

Reply:
it's just to ensure you send the mod channel joining order at the right time at client state, i don't like it, but it's preventive